peachjam/admin.py

```diff
@@ -1076,6 +1077,11 @@ class AttorneyAdmin(ImportExportMixin, admin.ModelAdmin):
     list_display = ("name", "description")
 
 
+@admin.register(DocumentMedia)
+class DocumentMediaAdmin(admin.ModelAdmin):
```

Comment:
This should be an inline in document admin, so it shows up as a tab (like topics, etc.)